src/components/ProjectSettingsTabs/VariablesTab/VariableRow.tsx

```diff
@@ -34,7 +34,10 @@ export const VariableRow: React.FC<
       </LeftColumn>
       <div>
         {variableValue}
-        {isPrivate}
+        <OptionRow>
```

Comment:
Can you please add screenshots of these changes and instructions for how to test it locally?

Reply:
(I'm reviewing this now anyway bc I'm in the PR zone 🎉  but should still do this ^ bc it's a good and helpful process)

Reply:
added a link!